docs/frontend-system/building-plugins/02-testing.md

```diff
@@ -6,8 +6,6 @@ sidebar_label: Testing
 description: Testing plugins in the frontend system
 ---
 
-> **NOTE: The new frontend system is in alpha and is only supported by a small number of plugins.**
-
 # Testing Frontend Plugins
```

Comment:
delete this too.

Then at the very top of the file, change `title: Frontend System Testing Plugins ` to `title: Testing Frontend Plugins`